diff --git a/src/schema.rs b/src/schema.rs
index 172b50f..da8692d 100644
--- a/src/schema.rs
+++ b/src/schema.rs
@@ -3,10 +3,10 @@
 diesel::table! {
     file_records (id) {
         id -> Integer,
         jid -> Nullable<Integer>,
         path -> Text,
         format -> Text,
-        modified_at -> Nullable<Timestamp>,
+        modified_at -> Nullable<TimestamptzSqlite>,
         size -> Nullable<BigInt>,
     }
 }
